installation-manager/src/main/java/org/wildfly/core/instmgr/logging/InstMgrLogger.java

```diff
@@ -39,12 +39,6 @@ public interface InstMgrLogger extends BasicLogger {
     @Message(id = 1, value = "No known attribute %s")
     OperationFailedException unknownAttribute(String asString);
 
-    @Message(id = 2, value = "Zip entry %s is outside of the target dir %s")
-    IOException zipEntryOutsideOfTarget(String entry, String target);
-
-    @Message(id = 3, value = "The Zip archive format is invalid. The '%s' directory cannot be found as a second-level entry within the unloaded Zip file.")
-    Exception invalidZipEntry(String directory);
-
```

Comment:
I'm looking at this commit by commit, so perhaps this is already handled in a later commit, but if not, once this is stable enough the message ids should be renumbered to eliminate any gaps.